src/components/DocumentationTopic/PrimaryContent/GenericContent.vue

```diff
@@ -31,19 +31,20 @@ export default {
   methods: {
     ...ContentNode.methods,
     addOnThisPageSections() {
```

Comment:
I assume this would also go away if we go with the Heading registration 🤔 @mportiz08

Reply:
It could, yeah—it would probably be better/simpler than this method. It will have the same issues with the live-editing scenario though I assume (as this one does).

Reply:
Yeah...

Reply:
Done - 63aa095c600efef747b029c6a5759214f1fe22df